skglm/tests/test_estimators.py

```diff
@@ -50,6 +52,7 @@
 C = 1 / alpha
 tol = 1e-10
 l1_ratio = 0.3
+groups = [20,30,10]
```

Comment:
ok this is definetly not going through flake8, because there should be spaces after the commas